packages/shared/src/lib/featureManagement.ts

```diff
@@ -6,6 +6,11 @@ export class Features {
     'Access all features',
   );
 
+  static readonly SignupButtonColor = new Features(
+    'signup_button_theme_color',
+    'primary',
+  );
```

Comment:
Think about this, something I forgot to mention is that we now also have `validOptions` as the third parameter here.
This should be an array of the options we allow to be valid.

```suggestion
  static readonly SignupButtonColor = new Features(
    'signup_button_theme_color',
    'primary',
    [
    'primary',
    'color1',
    'color2',
  ]
  );
```